airgo/refresh_data.go

```diff
@@ -0,0 +1,98 @@
+package airgo
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	airtable "github.com/mottaquikarim/go-airtable"
+	log "github.com/sirupsen/logrus"
+)
+
+type RefreshOpts struct {
+	ApiKey string
+	BaseId string
+	Conf   string
+	Dest   string
+}
+
+type Config struct {
+	Nav []ConfigItem
+}
+
+type ConfigItem struct {
+	Name     string `json:"name"`
+	Key      string `json:"key"`
+	Filename string `json:"filenameRoot"`
+	Viewname string `json:"viewname"`
+}
+
+func RefreshData(cfg RefreshOpts) error {
```

Comment:
From lines 31-43 we are making sure we are able to access the Airgo table? Not sure could you clarify this